user/build.xml

```diff
@@ -121,9 +121,7 @@
             ">
       <src path="super/com/google/gwt/emul/javaemul/internal" />
       <classpath>
-        <pathelement location="${gwt.tools.lib}/gss/2015-10-07/closure-stylesheets-library-20151007-rebased.jar"/>
```

Comment:
This shouldn't have still been here to begin with, but should probably land separately (ditto the tomcat line below).